release/main.py

```diff
@@ -31,6 +31,8 @@ def tag_all(manifest, tag, delete_if_exists=False):
     gtools = gittools.GitApiUtils()
     repos_with_tag = []
     for repo in manifest_contnet.keys():
+        if not repo.startswith("openshift/"):
+            continue
```

Comment:
Why skip if it's not openshift?
Add logging

Reply:
we don't have permission to tag it, adding logs

Reply:
@oshercc  I do have permission to tag it
so we should not skip it